backend/app/schemas/case.py

```
@@ -0,0 +1,16 @@
+from pydantic import BaseModel, ConfigDict
+from datetime import datetime
+from typing import Optional, List
```

Ruff (0.15.15)

[warning] `typing.List` is deprecated, use `list` instead

(UP035)